network/model.go

```diff
@@ -12,7 +12,28 @@ type CreateGameRequest struct {
 	PlayerIDs []model.PlayerID `json:"playerIDs"`
 }
 
+// TODO figure out the minimum info the client will need
+type CreateGameResponse struct {
+	Players         []model.Player                       `json:"players"`
```

Comment:
I'd say this shouldn't be a slice of `model.Player`, but instead `network.Player` which only has username (for now, but maybe someday have a user profile pic).